webui/react/src/pages/ExperimentDetails/ExperimentCheckpoints.settings.ts

```diff
@@ -39,65 +41,80 @@ export interface Settings extends InteractiveTableSettings {
   tableOffset: number;
 }
 
-const config: SettingsConfig = {
+const config: SettingsConfig<Settings> = {
   applicableRoutespace: '/checkpoints',
-  settings: [
-    {
+  settings: {
+    columns: {
       defaultValue: DEFAULT_COLUMNS,
-      key: 'columns',
       skipUrlEncoding: true,
       storageKey: 'columns',
-      type: {
-        baseType: BaseType.String,
-        isArray: true,
-      },
+      type: array(
```

Comment:
I think Trent removed it before, so `columns` should be `skipUrlEncoding: true`.